examples/07-EMIT/interference_gui.py.back

```diff
@@ -373,11 +431,21 @@ def populate_table(self):
         button.setEnabled(True)
         img_btn.setEnabled(True)
     
+    ###############################################################################
+    # GUI closing event
+    # ~~~~~~~~~~~~~~~~~
+    # Close AEDT if the GUI is closed.
     def closeEvent(self, event):
         emitapp.close_desktop()
 
+###############################################################################
+# Run GUI
+# ~~~~~~~
+# Launch the GUI. Ensure that window.show() and app.exec_() are 
+# uncommented to run the GUI. 
```

Comment:
```suggestion
# Launch the GUI. If you want to run the GUI, uncomment the ``window.show()`` and
# ``app.exec_()`` method calls. 
```

Reply:
@madelinecoleman1 It sounds like we need to move interference_classification.py out of the examples directory. We can move it to pyaedt\emit_core for now at least. Do you want to make that change with this PR or the testing PR? (either works for me)

Reply:
@jsalant22 sounds good, I'll move that file. I'll do it in this PR